migrations/0004_create_orders.up.sql

```diff
@@ -0,0 +1,21 @@
+create table orders (
+  id uuid primary key not null default uuid_generate_v4(),
+	created_at timestamp with time zone not null default current_timestamp,
+	updated_at timestamp with time zone not null default current_timestamp,
+	total_price text,
```

Comment:
should we really be using text here?

Reply:
also - shouldn't this and `merchant_id` / `status` be non nullable?

Reply:
Good point, I thought that we'd be storing string everywhere but based on our conversation I converted this type to be `numeric(28, 18)`
https://github.com/brave-intl/bat-go/pull/226/commits/bea43555003e75c8558bb5ace641ccfc237700f8